### PR TITLE
Feature/addlinter

### PR DESCRIPTION
**Pull request by mikihirpo**
Added linters and setup gitflow. Added
* Light house
* web hint
* style lint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,6 @@ My first repo of microverse
 Michael Hirpo
 mikaelhirpo@gmail.com
 
+## Company
+Microverse INC
+
```
Comment (ShahierNashaat) [inline README.md]: - [x] Kindly be noticed that the README file should be descriptive? AKA: Microverse README template should be used.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @mikihirpo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly be noticed that the Pull Request should have a descriptive title telling you what changes are inside.
- [ ] Kindly be noticed that the .gitignore file listing the` test.md` file and `node_modules` directory.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mikihirpo) [reply README.md]: what is the template?

Comment (ShahierNashaat) [reply README.md]: This is the [README Template](https://github.com/microverseinc/readme-template/blob/master/README.md).

Comment (mikihirpo) [reply README.md]: Thank you @ShahierNashaat  I have made the changes now and i have commited.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,65 @@
...
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
```
Comment (HAMMAS-SALEEM) [inline README.md:41]: - [ ] Please add your name here because you are the author of this repository. :+1:

File: README.md [hunk 3]
```diff
@@ -1,8 +1,65 @@
...
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (HAMMAS-SALEEM) [inline README.md:65]: - [ ] It's great that you have added a link to your MIT.md file which is your license file for this repository. But the problem is MIT.md file doesn't exist in your repo. :smile: Don't worry :relaxed:
- Create a MIT.md file in your main directory of this project.
- Copy the content from this [file](https://github.com/HAMMAS-SALEEM/Hello-Microverse/blob/master/MIT.md) and paste it into your MIT.md file.
- Last but not the least, add your name in place of Hammas Bin Saleem 

That's all :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# First of all, Welcome to Microverse :bouquet: :bouquet:

# Changes Requested :recycle:

almost there

## Hey @mikihirpo :wave: 
### Highlights :1st_place_medal:

Your project is working :clap:
Good commit history :heavy_check_mark:
Following Github Flow :heavy_check_mark:
Following HTML, CSS best practices :heavy_check_mark:
Using .gitignore file :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:

You have made a great effort on implementing some requested changes from the previous reviewer on your project :blush: and I can see that but you need to work on some issues to go to the next project but you are almost there. :+1:

## 1. 
title

- [ ] Your PR has a title that is not so descriptive enough. Because **Feature/addlinter** is not describing what are you doing in the project. PR title fully describes what is happening in this branch. You could use **Feature/Setup Hello Microverse Project** or **Feature/Initialize Hello Microverse Project** for your PR title. :+1:

## 2
### Your Summary
summary


### Descriptive Summary Example :heavy_check_mark:
clinton summary

## 3
hello requirements (copy)


- [ ] This is great that you are using the gitignore file for ignoring files. :+1: But you didn't add the test.md file here. Also, you didn't add a test.md file in your main directory. Please add test.md in gitignore and add test.md file in your main directory. :blush: 

### [Optional]
- [ ] You have a summary not descriptive enough for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking:

## [Optional]
- [ ] Your branch name is not descriptive enough. You are using feature/addlinter, you can use **feature/initialize-project** or **feature/setup-project** :+1: When you will change your branch name then this PR will be automatically closed then you will need to make a new PR. 
Best of Luck :+1:

_**All the requested changes with the [optional] tag are not crucial enough to prevent you from getting approval but it is highly recommended to you take them into consideration**_

### Once you implement all the changes you should be ready to submit your project for a re-review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**